<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mobile</class>
 <widget class="QWidget" name="mobile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculusSystem</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>281</width>
     <height>39</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #728495;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#544169;&quot;&gt;Original Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #648374;
border: 1px solid black;
font-size: 24px;
color: white;

</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Input Your Value</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::CursorMoveStyle::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QLabel" name="errorMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid red;
color: red;
background-color: (66, 66, 66);
font-size: 24px;</string>
   </property>
   <property name="text">
    <string>Wrong Input</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="firstBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>2 (binary)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8 (octal)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10 (ordinary)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16 (hexadecimal)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>281</width>
     <height>39</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #728495;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::WinPanel</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#544169;&quot;&gt;Resulting Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #648374;
border: 1px solid black;
font-size: 24px;
color: white;

</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="secondBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>2 (binary)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8 (octal)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10 (ordinary)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16 (hexadecimal)</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="convertButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #728495;
    border: 2px solid #111;
    border-radius: 8px;
    color: #111;
    font-size: 36px;
    height: 48px;
    padding: 0 25px;
    text-align: center;
    position: relative;
    outline: none;
}

QPushButton:hover {
    background-color: #4a5b6a; 
}

QPushButton:pressed {
    background-color: #4a5b6a; 
}

QPushButton::before {
    background-color: #111;
    border-radius: 8px;
    content: &quot;&quot;;
    height: 48px;
    left: 0;
    width: 100%;
    position: absolute;
    top: -2px;
    z-index: -1;
}

@media (min-width: 768px) {
    QPushButton {
        padding: 0 40px;
    }
}

</string>
   </property>
   <property name="text">
    <string>convert</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>640</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #728495;
    border: 2px solid #111;
    border-radius: 8px;
    color: #111;
    font-size: 36px;
    height: 48px;
    padding: 0 25px;
    text-align: center;
    position: relative;
    outline: none;
}

QPushButton:hover {
    background-color: #4a5b6a; 
}

QPushButton:pressed {
    background-color: #4a5b6a; 
}

QPushButton::before {
    background-color: #111;
    border-radius: 8px;
    content: &quot;&quot;;
    height: 48px;
    left: 0;
    width: 100%;
    position: absolute;
    top: -2px;
    z-index: -1;
}

@media (min-width: 768px) {
    QPushButton {
        padding: 0 40px;
    }
}

</string>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="languageBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>780</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Русский</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
